From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: IslandZed <islandzed@local>
Date: Thu, 7 May 2026 00:00:00 +0000
Subject: [PATCH 3/9] scrollbar-style


diff --git a/crates/editor/src/element.rs b/crates/editor/src/element.rs
index 23d90c6e8a..d1bf8954e3 100644
--- a/crates/editor/src/element.rs
+++ b/crates/editor/src/element.rs
@@ -7005,11 +7005,11 @@ impl EditorElement {
                         };
                         window.paint_quad(quad(
                             thumb_bounds,
-                            Corners::default(),
+                            Corners::all(Pixels::MAX).clamp_radii_for_quad_size(thumb_bounds.size),
                             scrollbar_thumb_color,
-                            scrollbar_edges,
-                            cx.theme().colors().scrollbar_thumb_border,
-                            BorderStyle::Solid,
+                            Edges::default(),
+                            Hsla::transparent_black(),
+                            BorderStyle::default(),
                         ));
 
                         if any_scrollbar_dragged {
diff --git a/crates/ui/src/components/scrollbar.rs b/crates/ui/src/components/scrollbar.rs
index 4a30f60176..ef7a38d2ef 100644
--- a/crates/ui/src/components/scrollbar.rs
+++ b/crates/ui/src/components/scrollbar.rs
@@ -359,7 +359,7 @@ impl ScrollbarStyle {
     pub const fn to_pixels(&self) -> Pixels {
         match self {
             ScrollbarStyle::Regular => px(6.),
-            ScrollbarStyle::Editor => px(15.),
+            ScrollbarStyle::Editor => px(6.),
         }
     }
 }
